content/en/core/releases/4.4.0.md

```diff
@@ -0,0 +1,103 @@
+- [#8179](https://github.com/medic/cht-core/issues/8179): nginx and haproxy should respect `Accept` headers when responding with error templates
+- [#8390](https://github.com/medic/cht-core/issues/8390): Align "dialog" components with Material design guidelines
+- [#8442](https://github.com/medic/cht-core/issues/8442): Telemetry geolocation:failure events for geolocation should be unique
+- [#8498](https://github.com/medic/cht-core/issues/8498): Remove targets disable admin option
+- [#8506](https://github.com/medic/cht-core/issues/8506): Update browser support notice to match official versions
+- [#8511](https://github.com/medic/cht-core/issues/8511): Update configuration deployment action to use latest cht-conf
+
+### Security fixes
+
+- [#8443](https://github.com/medic/cht-core/issues/8443): Security issue 8156
+
+### Performance improvements
+
+- [#8238](https://github.com/medic/cht-core/issues/8238): Figure out a different way to use purged docs cache
+- [#8451](https://github.com/medic/cht-core/issues/8451): Add support for HTTP/2 in nginx 
+
+### Bug fixes
+
+- [#6180](https://github.com/medic/cht-core/issues/6180): Any text containing back ticks looks weird in the webapp
+- [#8081](https://github.com/medic/cht-core/issues/8081): `/api/v2/export/contacts` does not return useful contact type data for custom hierarchy
+- [#8243](https://github.com/medic/cht-core/issues/8243): Can't turn off permissions for `national_admin` role in admin app
+- [#8490](https://github.com/medic/cht-core/issues/8490): API crashes when CouchDB is overloaded
+- [#8527](https://github.com/medic/cht-core/issues/8527): Unable to build nginx with symlinks in some environments
+
+### Technical improvements
+
+- [#6132](https://github.com/medic/cht-core/issues/6132): Files containing tests should use the .spec.js extension
+- [#6289](https://github.com/medic/cht-core/issues/6289): Upgrade to CouchDB v3
+- [#8306](https://github.com/medic/cht-core/issues/8306): Replace grunt with npm scripts
+- [#8414](https://github.com/medic/cht-core/issues/8414): Flaky test: sms-gateway api returns list and updates state
+- [#8459](https://github.com/medic/cht-core/issues/8459): Update nginx to latest 
+- [#8469](https://github.com/medic/cht-core/issues/8469): Scalability suite fails because Jmeter version no longer exists
+- [#8494](https://github.com/medic/cht-core/issues/8494): Update lodash
+- [#8516](https://github.com/medic/cht-core/issues/8516): Remove unused /api/auth/ endpoint
+- [#8518](https://github.com/medic/cht-core/issues/8518): Encode parameters being used in credentials url
+- [#8522](https://github.com/medic/cht-core/issues/8522): Flaky test: More Options Menu - Offline User all permissions disabled
+- [#8524](https://github.com/medic/cht-core/issues/8524): Remove sub-scripts from package.json
+- [#8533](https://github.com/medic/cht-core/issues/8533): Builds failing pushing build history to git repo
```

Comment:
```suggestion
- [#8533](https://github.com/medic/cht-core/issues/8533): Builds failing pushing build history to git repo
```
I read it 3 times processing the meaning 😅 what about: `Ignore unimportant step failures in CI build`